public/locales/zh/translation.json

```diff
@@ -0,0 +1,110 @@
+  "Error code": "错误代码",
+  "Location": "位置",
+  "Action index": "动作索引",
+  "Line": "行",
+  "Close": "关闭",
+  "Loading failed": "加载失败",
+  "Retry": "重试",
+  "Filter by Author": "按作者筛选",
+  "Clear Author filter": "清除作者筛选",
+  "Sort by": "排序方式",
+  "Sort field": "排序字段",
+  "Title": "标题",
+  "Size": "大小",
+  "Last Updated": "最后更新",
+  "Sort Ascending": "升序排序",
+  "Sort Descending": "降序排序",
+  "Search": "搜索",
+  "Author": "作者",
+  "Ported by": "移植者",
+  "Version": "版本",
+  "Type": "类型",
+  "Last update": "最后更新",
+  "Read Description": "阅读描述",
+  "On Shelf": "在书架上",
+  "Add to shelf": "添加到书架",
+  "Notice": "通知",
+  "Run": "运行",
+  "OK": "确定",
+  "Cancel": "取消",
+  "Resume": "继续",
+  "Restart": "重新开始",
+  "Saves": "保存",
+  "Save": "保存游戏",
+  "Load": "加载游戏",
+  "Clear": "清除",
+  "Preferences": "首选项",
+  "Save game": "保存游戏",
+  "Load game": "加载游戏",
+  "Empty": "空",
+  "Division by zero!": "除以零！",
+  "Type mismatch!": "类型不匹配！",
+  "Stack overflow!": "栈溢出！",
+  "Too many items in expression!": "表达式中的项过多！",
+  "Can't load file!": "无法加载文件！",
+  "Game not loaded!": "游戏未加载！",
+  "Sign [:] not found!": "找不到符号[:]！",
+  "Can't add file!": "无法添加文件！",
+  "Can't add action!": "无法添加动作！",
+  "Sign [=] not found!": "找不到符号[=]！",
+  "Location not found!": "找不到位置！",
+  "[end] not found!": "找不到[end]！",
+  "Label not found!": "找不到标签！",
+  "Incorrect variable name!": "错误的变量名！",
+  "Quote not found!": "找不到引用！",
+  "Bracket not found!": "找不到括号！",
+  "Brackets not found!": "找不到括号！",
+  "Syntax error!": "语法错误！",
+  "Unknown action!": "未知动作！",
+  "Incorrect arguments' count!": "参数数量不正确！",
+  "Can't add object!": "无法添加对象！",
+  "Can't add menu item!": "无法添加菜单项！",
+  "Too many variables!": "变量过多！",
+  "Regular expression error!": "正则表达式错误！",
+  "Code not found!": "找不到代码！",
+  "Loop condition not found!": "找不到循环条件！",
+  "Unknown error!": "未知错误！",
+  "Config not found": "找不到配置",
+  "Game not found": "找不到游戏",
+  "Game file not found": "找不到游戏文件",
+  "Game files not found inside archive": "存档中找不到游戏文件",
+  "Failed to load game": "加载游戏失败",
+  "Failed to load catalog": "加载目录失败",
+  "Unsupported archive format": "不支持的存档格式",
+  "File not found": "找不到文件",
+  "Open game": "打开游戏",
+  "Quit": "退出",
+  "Game has been updated in catalog": "目录中的游戏已更新",
+  "Update on shelf": "更新书架",
+  "Game shelf is empty": "游戏书架为空",
+  "Volume": "音量",
+  "Mute": "静音",
+  "Unmute": "取消静音",
+  "{{ name }} added to shelf": "{{ name }} 已添加到书架",
+  "Locale": "地区",
+  "Hotkeys": "热键",
+  "Action #1": "动作 #1",
+  "Action #2": "动作 #2",
+  "Action #3": "动作 #3",
+  "Action #4": "动作 #4",
+  "Action #5": "动作 #5",
+  "Action #6": "动作 #6",
+  "Action #7": "动作 #7",
+  "Action #8": "动作 #8",
+  "Action #9": "动作 #9",
+  "Single action": "单一动作",
+  "Quicksave": "快速保存",
+  "Quickload": "快速加载",
+  "Increase volume": "增加音量",
+  "Decrease volume": "减少音量",
+  "n/a": "不适用",
+  "Aero games": "Aero游戏",
+  "Game with id {{catalogId}} not found in catalog": "目录中找不到ID为 {{catalogId}} 的游戏",
+  "Loading game from catalog...": "从目录加载游戏中…"
```

Comment:
You missed one key in translation that was added recently - `"Theme {{name}} not found"`